mlflow/store/db/utils.py

```diff
@@ -184,6 +186,22 @@ def _upgrade_db_initialized_before_mlflow_1(engine):
     command.stamp(config, "base")
 
 
+def create_sqlalchemy_engine_with_retry(db_uri):
+    attempts = 0
+    while True:
+        attempts += 1
+        engine = create_sqlalchemy_engine(db_uri)
+        try:
+            sqlalchemy.inspect(engine)
+            return engine
+        except:
+            if attempts < MAX_RETRY_COUNT:
```

Comment:
Can we add a warning log with the exception and indicate that we're trying again?

Reply:
Sure, I added it.